admin_guide/cluster_capacity.adoc

```diff
@@ -0,0 +1,256 @@
+[[admin-guide-cluster-capacity]]
+= Analyzing Cluster Capacity
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As a cluster administrator, you can use the cluster capacity tool to monitor a
+node's available resources in order to increase the current resources before
+they become exhausted, and to ensure any future pods can be scheduled. This
+capacity comes from an individual node host in a cluster, and includes CPU,
+memory, disk space, and others.
+
+The cluster capacity tool simulates a sequence of scheduling decisions to
+determine how many instances of an input pod can be scheduled on the cluster
+before it is exhausted of resources in order to provide a more accurate
+estimation.
+
+[NOTE]
+====
+The remaining allocatable capacity is a rough estimation, because it does not
+count all of the resources being distributed among nodes. It analyzes only the
+remaining resources and estimates the available capacity that is still
+consumable in terms of a number of instances of a pod with given requirements
+that can be scheduled in a cluster.
+
+Also, pods might only have scheduling support on particular sets of nodes based
+on its selection and affinity criteria. As a result, the estimation of which
+remaining pods a cluster can schedule can be difficult.
+====
+
+The cluster capacity analysis tool can be run either as a stand-alone utility
+from the command line, or as a job in a pod inside an {product-title} cluster in
+order for the pod to be run multiple times, such as in a
+xref:../dev_guide/cron_jobs.adoc#dev-guide-cron-jobs[cron job].
```

Comment:
Cron job has its own kind 

```
apiVersion: batch/v2alpha1
kind: CronJob
```
Where here we are using just `kind: Job` as an example.